src/lein_monolith/task/each.clj

```diff
@@ -94,11 +94,11 @@
 
 (defn- init-project
   "Initialize the given subproject to prepare to run a task in it."
-  [subproject]
+  [subproject ctx]
   (locking init-lock
     (config/debug-profile "init-subproject"
       (project/init-project
-        (plugin/add-middleware subproject)))))
+        (plugin/apply-middleware subproject (:monolith ctx))))))
```

Comment:
Following the above, this would become:
```suggestion
        (plugin/apply-middleware
          (get-in ctx [:subprojects target])
          (:monolith ctx))))))
```